docs2/site/docs/getting-started/authorization.md

```diff
@@ -1,114 +1,41 @@
 # Authorization
 
-> See the [Authorization](https://github.com/graphql-dotnet/authorization) project for a
-> more in depth implementation of the following idea. Keep in mind that alongside this
-> project there is a similar [Authorization.AspNetCore](https://github.com/graphql-dotnet/server/tree/develop/src/Authorization.AspNetCore)
-> project specifically for ASP.NET Core apps.
+The GraphQL operation authorization problem is out of scope of GraphQL.NET project but we
+provide some [extension methods](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL/AuthorizationExtensions.cs)
+to configure authorization requirements to fields, types and other shema elements.
 
-You can write validation rules that will run before a query is executed. You can use this
-pattern to check that the user is authenticated or has permissions for a specific field.
-This example uses the `Metadata` dictionary available on Fields to set permissions per field.
+You can write validation rules that will run before a GraphQL document is executed. You
+can use this pattern to check that the user is authenticated or has permissions for a
+specific field or type.
 
-```csharp
-public class MyGraphType : ObjectGraphType
-{
-  public MyGraphType()
-  {
-    this.RequirePermission("READ_ONLY");
-    Field(x => x.Secret).RequirePermission("Admin");
-  }
-}
-```
+See the [Authorization](https://github.com/graphql-dotnet/authorization) project for an
+implementation of authorization framework on top of GraphQL.NET. Keep in mind that alongside
+this project there is a similar [Authorization.AspNetCore](https://github.com/graphql-dotnet/server/tree/develop/src/Authorization.AspNetCore)
```

Comment:
I think this link should point to the home of the server repo, which should contain documentation within the readme file, or to a dedicated documentation site.  Perhaps it could point to a documentation file within the repo.  But essentially, it should point to other documentation/samples so people can learn how to implement it, not to source code.